pilot/pkg/proxy/envoy/v2/discovery.go

```diff
@@ -85,7 +84,11 @@ func NewDiscoveryServer(env model.Environment, generator *v1alpha3.ConfigGenerat
 		ConfigGenerator: generator,
 	}
 
-	if len(periodicRefreshDuration) > 0 {
+	envOverride := os.Getenv("V2_REFRESH")
+	if len(envOverride) > 0 {
+		periodicRefreshDuration, _ = time.ParseDuration(envOverride)
```

Comment:
I assume you're ignoring the error, since we'll just skip the `if` below? Add a comment to make it clear.

Reply:
done.